framework/web/UrlManager.php

```diff
@@ -117,14 +117,20 @@ class UrlManager extends Component
      *
      * After the UrlManager object is created, if you want to change this property,
      * you should only assign it with a cache object.
-     * Set this property to false if you do not want to cache the URL rules.
+     * Set this property to `false` if you do not want to cache the URL rules.
      */
     public $cache = 'cache';
     /**
      * @var array the default configuration of URL rules. Individual rule configurations
      * specified via [[rules]] will take precedence when the same property of the rule is configured.
      */
     public $ruleConfig = ['class' => 'yii\web\UrlRule'];
+    /**
+     * @var UrlNormalizer|array|boolean the configuration for [[UrlNormalizer]] used by this UrlManager.
+     * If `false` normalization will be skipped. If `true`, normalizer will use default settings.
+     * @since 2.0.10
+     */
+    public $normalizer = true;
```

Comment:
`['class' => 'yii\web\UrlNormalizer']` will not work for UrlRule - each rule will get then own instance of normalizer. I'd like to avoid different syntax for configuring normalizer for UrlRule and UrlManager. 
Also `true|false` is IMO more intuitive, because you can read it as "normalizer is enabled/disabled". `$normalizer = null` looks like "there is no normalizer, so no normalization should be done".